static/js/badge.js

```diff
@@ -0,0 +1,39 @@
+const classMap = {
+    "hanged": "haczyk_key_on",
+    "empty": "haczyk_key_off"
+};
+
+function handleError(){
+    const elements = document.querySelectorAll('.haczyk_right');
+    elements.forEach(element => {
+        element.className='haczyk_right haczyk_key_error';
+    })
+    console.log("fetch_error");
+}
+
+function updateButtonState(element_id, current_state){
+    element_id.className='haczyk_right';
+    if (classMap.hasOwnProperty(current_state)) {
+        element_id.classList.add(classMap[current_state]);
+    } else {
+        element_id.classList.add("haczyk_key_error"); 
+    }
+}
+
+function getState(){
+    fetch('hangwatch.knr.edu.pl/hooks')
```

Comment:
Popełniłeś tutaj błąd. 
```suggestion
    fetch('https://hangwatch.knr.edu.pl/hooks')
```